pkg/systemd/overview.jsx

```diff
@@ -133,8 +133,8 @@ class LoginMessages extends React.Component {
         if (!last_login_message && !fail_count_message && !last_fail_message)
             return null;
 
-        const last_log_item = <p id='last-login'><b>{_("Last login:")}</b> {last_login_message}</p>;
-        const last_fail_item = <p id='last-failed-login'><b>{_("Last failed login:")}</b> {last_fail_message}</p>;
+        const last_log_item = <span id='last-login'><strong>{_("Last login:")}</strong> {last_login_message}</span>;
+        const last_fail_item = <span id='last-failed-login'><strong>{_("Last failed login:")}</strong> {last_fail_message}</span>;
```

Comment:
We should _not_ have a paragraph tag `<p>` inside a heading. An inline element like a `<span>` is more appropriate for this. Additionally, we should use semantic elements like `<strong>` instead of meaningless presentation-only elements like `<b>`. (It looks the same, but has meaning.)

This change was an attempt to fix the alignment of the alert information, but it didn't fix it. I adjusted the CSS after and _that_ worked. However, the `<p>` in an `<h4>` is still an issue, so I included the fix here as it would (probably) still impact the alignment, and it's still definitely wrong.

Reply:
No, that's still needed. H4 is bold by default and in some alerts, it's not normal color text (such as a warning). We have a `<strong>` inside an `<h4>` for the log in alert, for example. By removing this block, it makes the entire string emboldened and the heading uses RedHatDisplay instead of RedHatText. (in others, such as alerts, it affects the color too. It doesn't in this case.)

I just tested this.

Without the block:

![Screenshot 2021-09-06 at 10-45-06 Overview - garrett Bolt](https://user-images.githubusercontent.com/10246/132188102-764de7ae-c19f-478e-b04e-e781b8d6dfb5.png)

With the block:

![Screenshot 2021-09-06 at 10-45-59 Overview - garrett Bolt](https://user-images.githubusercontent.com/10246/132188070-5ea75153-a1ea-4502-96bc-69aa29397663.png)

Reply:
Oh, looking at the screenshots, the color override is also in effect here too. (Dark blue to black.)